lib/resty/lmdb/transaction.lua

```diff
@@ -71,6 +71,25 @@ function _M.begin(hint)
 end
 
 
+local normalize_key
+do
+    local sha256 = assert(require("resty.sha256").new())
```

Comment:
Should we just use `resty.openssl`? Not a big deal, and having dependency outside core OpenResty is a consideration too. Just thinking that main user is Kong, so should we just go ahead and use crypto lib that we have otherwise standardized to.

Reply:
I also think about it. But `resty.openssl` is not bundled in OpenResty, it do introduce extra dependency.

Reply:
With an offline discussion, we decide that we will not use `resty.openssl`.